src/applications/caregivers/config/form.js

```diff
@@ -296,37 +346,45 @@ const formConfig = {
       },
     },
     secondaryCaregiversChapter: {
-      title: 'SECONDARY CAREGIVERS',
+      title: 'Secondary Family Caregivers Information',
       pages: {
-        secondaryOneCaregiverIntro: {
+        secondaryCaregiverOneIntro: {
           path: 'secondaryOne-caregiver-intro',
-          title: 'Secondary Caregiver Information',
+          title: ' ',
           uiSchema: {
-            'ui:description': SecondaryCaregiverInfo,
-            [primaryCaregiverFields.hasSecondaryOneCaregiverView]: hasSecondaryOneCaregiverUI,
+            'ui:description': SecondaryCaregiverInfo({ additionalInfo: true }),
+            [primaryCaregiverFields.hassecondaryCaregiverOneView]: hasSecondaryCaregiverOneUI,
           },
           schema: {
             type: 'object',
             properties: {
-              [primaryCaregiverFields.hasSecondaryOneCaregiverView]: {
+              [primaryCaregiverFields.hassecondaryCaregiverOneView]: {
                 type: 'boolean',
               },
             },
           },
         },
-        secondaryOneCaregiver: {
+        secondaryCaregiverOne: {
           path: 'secondaryOne-caregiver-1',
-          title: 'Secondary Caregiver Information',
-          depends: formData => hasSecondaryOneCaregiver(formData),
+          title: 'Secondary Family Caregiver One Information',
+          depends: formData => hasSecondaryCaregiverOne(formData),
           uiSchema: {
-            'ui:description': SecondaryCaregiverInfo,
+            'ui:description': SecondaryCaregiverInfo({
+              pageTitle: 'Secondary Caregiver One Information',
+            }),
             // secondaryOne UI
-            [secondaryCaregiverFields.secondaryOne.fullName]:
-              secondaryCaregiverUI.secondaryOne.fullNameUI,
-            [secondaryCaregiverFields.secondaryOne.ssn]:
-              secondaryCaregiverUI.secondaryOne.ssnUI,
-            [secondaryCaregiverFields.secondaryOne.dateOfBirth]: dateOfBirthUI,
-            [secondaryCaregiverFields.secondaryOne.gender]: genderUI,
+            [secondaryCaregiverFields.secondaryOne.fullName]: fullNameUI(
+              'Secondary One Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryOne.ssn]: ssnUI(
+              'Secondary One Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryOne.dateOfBirth]: dateOfBirthUI(
+              'Secondary One Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryOne.gender]: genderUI(
+              'Secondary One Family Caregiver',
+            ),
           },
```

Comment:
Maybe I'm missing something?

Reply:
Silly stakeholder requirements :( had to change gender to sex and add "unknown"